experimental/incremental/task.go

```diff
@@ -16,15 +16,32 @@ package incremental
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"iter"
+	"os"
+	"runtime"
 	"runtime/debug"
 	"slices"
 	"sync"
 	"sync/atomic"
 
+	"golang.org/x/sync/semaphore"
+
 	"github.com/bufbuild/protocompile/experimental/report"
 )
 
+// Set this to true to enable debug printing.
+const debugIncremental = false
+
+// ErrNilQuery is set as the Fatal error for the result of executing a nil
+// query.
```

Comment:
nit: move this to the actual exported `ErrNilQuery` instead of it being for the entire block of vars.